src/executors/default.yml

```diff
@@ -8,4 +8,4 @@ parameters:
   image:
     description: Specify Docker image for executor
     type: string
-    default: hashicorp/terraform:0.14.11 # update commands/install when updating this
+    default: hashicorp/terraform:1.0.0 # update commands/install when updating this
```

Comment:
We should really change this to specify the version of the Docker image, rather than the entire image. This parameter is completely redundant.